quickstart/prerequisites.md

```diff
@@ -5,7 +5,7 @@
 All worker nodes must satisfy the following requirements:
 
 * **x86-64** CPU cores with SSE4.2 instruction support
-* Linux kernel **5.13 or higher** with the following modules loaded:
+* Linux kernel **5.15.74 or higher** with the following modules loaded:
```

Comment:
Any specific benefit that we get with this? I'm asking because, Linux distributions (both cloud and on-premise) usually take time to bump up their kernel versions.

Reply:
I think it's optics.  It doesn't seem appropriate to be recommending a version in any context, which is EOL, and on which we no longer test.  As of yesterday, system testing moved to 5.15 as their minimum deployed kernel

Reply:
I don't think we should do this, that version is VERY recent, might not be widely available yet.
Also, should this be a requirement or a recommendation?
Another point raise by @blaisedias, we haven't even tested this version, is it a good idea to recommend what we've never tested?

Reply:
As per the engineering team discussion, we have decided to make couple of changes here.
1. Use the e2e-tested version i.e. 5.15
2. Mention the phrase "Recommended and Tested" instead of a hard pre-requisite.
@anupriya0703 can you make the above changes?

Reply:
Sure, will be making the above changes.